2022q4/pantheon.adoc

```diff
@@ -0,0 +1,40 @@
+=== Pantheon desktop on FreeBSD
+
+Links: +
+link:https://elementary.io/[elementary OS] URL: link:https://elementary.io/[https://elementary.io] +
+link:https://codeberg.org/olivierd/freebsd-ports-elementary[Development repository] URL: link:https://codeberg.org/olivierd/freebsd-ports-elementary[https://codeberg.org/olivierd/freebsd-ports-elementary] +
+
+Contact: Olivier Duchateau <duchateau.olivier@gmail.com>
+
+The Pantheon desktop environment is designed for elementary OS.
+It builds on GNOME technologies (such as Mutter, GTK 3 and 4) and it is written in Vala. The goal is to have a new desktop for users.
+
+*13.1-RELEASE or higher is required*, because several core components depend of `deskutils/xdg-desktop-portal`.
+
+The repository contains Mk/Uses framework `elementary.mk`, official applications, and curated ports which depend of `x11-toolkits/granite`.
+
+Since the lastest report, we continue to keep up to date its related ports, especially:
+
+* `deskutils/elementary-calendar` update to link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=267797[6.1.2]
+* `deskutils/iconbrowser`
+* `graphics/elementary-photos`
+* `math/elementary-calculator`
+* `multimedia/elementary-videos`
+* `x11/elementary-terminal`
+* `x11-themes/gnome-icons-elementary`
+* `x11-toolkits/granite7`
+
+Power manager plugins for top panel (wingpanel) and control center (switchboard) are finished.
+
+A new switchboard plugin is also available, `net/switchboard-plug-sharing`. Ported *Rygel*, GNOME UPnP/DLNA services.
+
+Submitted others patches for low level libraries such as:
+
+* `print/cups-pk-helper` update to link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=266067[0.2.7] required by `print/switchboard-plus-printers`
+* `devel/libgee` update to link:https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=266585[0.20.6] heavily uses by the desktop
+* `sysutils/polkit` update to 122 link:https://reviews.freebsd.org/D37137[D37137]
+* `sysutils/accountsservice` update to 22.08.8 link:https://reviews.freebsd.org/D37679[D37679]
+
+==== Open tasks
```

Comment:
I think you need an extra linebreak (empty line) between heading and bullet list.